forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/ConfirmationCallbackFragment.java

```diff
@@ -45,4 +45,76 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         return view;
     }
 
+/*
```

Comment:
Are you intentionally leaving this code commented? Should we leave an explanation why this commented out?